blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClient.scala

```diff
@@ -103,17 +103,18 @@ object BlazeClient {
                 case Some(stage) => F.async[TimeoutException](stage.init)
                 case None => F.never[TimeoutException]
               }
-              val res = next.connection
+              val res: F[Resource[F, Response[F]]] = next.connection
```

Comment:
This might end up being nice in 1.0, as we move toward responses being resources.